src/variants/banana-dark.scss

```
@@ -26,7 +26,7 @@
 @define-color accent_color_700 @BANANA_700;
 @define-color accent_color_900 @BANANA_900;
 
-@define-color accent_color @BANANA_500;
+@define-color accent_color mix (@BANANA_500, @BANANA_700, 0.25);
```

nitpick, but typically in (S)CSS, functions don't have whitespace before parens

```suggestion
@define-color accent_color mix(@BANANA_500, @BANANA_700, 0.25);
```